packages/warriorjs-cli/src/parseArgs.js

```diff
@@ -35,6 +35,11 @@ function parseArgs(args) {
         describe: 'Skip user input',
         type: 'boolean',
       },
+      m: {
+        alias: 'silent',
+        describe: 'Skip displaying game logs',
```

Comment:
What we are suppressing is the play log, what do you think we change this to "Suppress play log"?